tests/integration/infrastructure/rpc/test_simulator_sessions.py

```
@@ -0,0 +1,440 @@
+        model="model",
+    )
+
+    assert result == {"address": "0xabc"}
+    registry_instance.update_validator.assert_awaited_once()
+    validator_arg = registry_instance.update_validator.await_args.args[0]
+    assert validator_arg.address == "0xabc"
+    assert validator_arg.stake == 42
+
+
+@pytest.mark.asyncio
+async def test_delete_validator_uses_request_session(monkeypatch):
```

Ruff (0.14.0)

Unused function argument: `monkeypatch`

(ARG001)